### PR TITLE
 Fixes#351 Badgeholder Designation to be drawn from CSV 

### LINKED ISSUE #351: Add options like speaker, guest etc. to be drawn from CSV instead of hardcoding it in the image
Currently the default images are showing the name and logo of the event (here FOSSASIA) and the speaker, guest, camera crew etc.

Instead implement as follows:
* Add item for role into CSV or manual box, e.g. Hong Phuc,Dang,Organiser,@hpdang,FOSSASIA
* Change images and take out FOSSASIA logo and roles, e.g. speaker
* Also rename background image to color and theme item. Use e.g. "gray_futuristic", "red_futuristic"

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #351 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Now User can input the designation either in CSV or Manually entering the details.
- https://infinite-peak-45789.herokuapp.com/

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 								<section id="text-input" style="display:none">
 									<div class="form-group">
 										<label for="textArea"> Enter One Name per Line </label>
-										<textarea class="form-control" type="text" placeholder="Eg. Hong Phuc,Dang,@hpdang,FOSSASIA " id="textArea" name="csv"></textarea>
+										<textarea class="form-control" type="text" placeholder="Eg. Hong Phuc,Dang,Organiser,@hpdang,FOSSASIA " id="textArea" name="csv"></textarea>
```
Comment (vaibhavsingh97) [inline app/templates/index.html]: Remove `,` from name `Hong Phuc Dang`

**Comment by vaibhavsingh97** [comment]
Look like Travis CI has some issue :(
also, open new issue for `Change images and take out FOSSASIA logo`, refrencing #351

File: app/templates/index.html (hunk 1 above)
Comment (ParthS007) [reply app/templates/index.html]: @vaibhavsingh97  Name appears in two lines as before.

**Comment by ParthS007** [comment]
@vaibhavsingh97  I will remove FOSSASIA logo in #354 .

**Comment by ParthS007** [comment]
image

**Comment by ParthS007** [comment]
@djmgit @harshit98 Please Review.Thanks

**Comment by mohdomama** [comment]
Great Work @ParthS007 :+1:

File: app/templates/index.html (hunk 1 above)
Comment (gabru-md) [reply app/templates/index.html]: @vaibhavsingh97 Name appears in two lines. You can refer to the issue where Mario explicitly stated the format to be used.

**Pull request merged**